.github/labeler.yml

```diff
@@ -13,13 +19,13 @@ Particles: plasmapy/particles/**/*
 # Add Documentation label to any change in docs but notebooks
 Documentation:
 - any: [docs/**/*]
-  all: ['!docs/notebooks/**/*']
+  all: ['!docs/notebooks/**/*', '!docs/contributing/*']
```

Comment:
I'm hoping I got the YAML right on this one.

Reply:
You can copy-paste the whole document into a validator such as http://www.yamllint.com/ - it says this is okay :)